endpoints/api_config.py

```diff
@@ -152,6 +152,9 @@ def _Enum(docstring, *names):
 
 AUTH_LEVEL = _Enum(_AUTH_LEVEL_DOCSTRING, 'REQUIRED', 'OPTIONAL',
                    'OPTIONAL_CONTINUE', 'NONE')
+_AUTH_LEVEL_WARNING = ("Due to a design error, auth_level has never actually been functional. "
+                       "It will likely be removed and replaced by a functioning alternative "
```

Comment:
Do we actually have plans to provide this functionality?

Reply:
Eh, I actually considered doing so in this release. It's not too hard. But I decided to ship the release with this warning first to give folks some time to adjust.

Reply:
So, I'd go with "probably".